elastic/beats
# Add `job` label by default when using Prometheus collector

This PR adds `job` label by default when using prometheus collector. So when using raw Prometheus collector it will always be `prometheus`. 

When using lightweight modules it is the name of the lightweight module ie `cockroachdb`

Closes #12739
### Testing:
1. Try the module with `metrics_path: /metrics` in the configuration and collect metrics from a `node-exporter` directly.
2. Try the module with `metrics_path: '/federate'` and collect metrics from Prometheus server.

In both cases label `job` should be present.
https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-metricset-prometheus-collector.html

Extra: Try the same with `cockroachdb` lightweight module. Unitest's testdata cover this case too.
